warble/functions.cc

```diff
@@ -69,10 +70,17 @@ bool Warble(Database* db, Any req, Any* rep) {
   } else {
     LOG(INFO) << "no parent id found " << parent_id;
   }
+  std::istringstream iss(text);
+  std::vector<std::string> result{std::istream_iterator<std::string>(iss), {}};
+  for (auto r : result) {
+    if (r[0] == '#') {
+      db->put("_hashtag_" + r, w_as_string);
+    }
+  }
```

Comment:
Are there advantages to using a stringstream? I feel this could be made simpler using a C string.